2022-01-20 16:27:41,065 INFO    MainThread:6724 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220120_162741-3srcro79\logs\debug.log
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d02_train\wandb\run-20220120_162741-3srcro79\logs\debug-internal.log
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002DDA37F0730>
2022-01-20 16:27:41,066 INFO    MainThread:6724 [wandb_init.py:init():404] calling init triggers
2022-01-20 16:27:41,067 INFO    MainThread:6724 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-20 16:27:41,067 INFO    MainThread:6724 [wandb_init.py:init():449] starting backend
2022-01-20 16:27:41,067 INFO    MainThread:6724 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-20 16:27:41,071 INFO    MainThread:6724 [backend.py:ensure_launched():199] starting backend process...
2022-01-20 16:27:41,388 INFO    MainThread:6724 [backend.py:ensure_launched():204] started backend process with pid: 23052
2022-01-20 16:27:41,389 INFO    MainThread:6724 [wandb_init.py:init():458] backend started and connected
2022-01-20 16:27:41,393 INFO    MainThread:6724 [wandb_run.py:_label_probe_notebook():917] probe notebook
2022-01-20 16:27:41,394 INFO    MainThread:6724 [wandb_run.py:_label_probe_notebook():927] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-01-20 16:27:41,394 INFO    MainThread:6724 [wandb_init.py:init():519] updated telemetry
2022-01-20 16:27:41,402 INFO    MainThread:6724 [wandb_init.py:init():549] communicating current version
2022-01-20 16:27:42,786 INFO    MainThread:6724 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-20 16:27:42,786 INFO    MainThread:6724 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-20 16:27:43,004 INFO    MainThread:6724 [wandb_init.py:init():592] starting run threads in backend
2022-01-20 16:27:43,971 INFO    MainThread:6724 [wandb_run.py:_console_start():1816] atexit reg
2022-01-20 16:27:43,971 INFO    MainThread:6724 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-20 16:27:43,972 INFO    MainThread:6724 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-20 16:27:43,972 INFO    MainThread:6724 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-20 16:27:43,973 INFO    MainThread:6724 [wandb_init.py:init():619] run started, returning control to user process
2022-01-20 16:27:44,987 INFO    MainThread:6724 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/3srcro79
2022-01-20 16:27:45,048 INFO    MainThread:6724 [jupyter.py:save_history():429] not saving jupyter history
2022-01-20 16:27:45,048 INFO    MainThread:6724 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-01-20 16:27:45,048 INFO    MainThread:6724 [wandb_init.py:_jupyter_teardown():303] cleaning up jupyter logic
2022-01-20 16:27:45,048 INFO    MainThread:6724 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2022-01-20 16:27:45,049 INFO    MainThread:6724 [wandb_run.py:_restore():1758] restore
2022-01-20 16:27:45,480 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 618
}

2022-01-20 16:27:46,124 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 618
}

2022-01-20 16:27:46,336 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 618
}

2022-01-20 16:27:47,018 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 4309
}

2022-01-20 16:27:47,119 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 4309
}

2022-01-20 16:27:47,221 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 618
  total_bytes: 4309
}

2022-01-20 16:27:47,323 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4283
  total_bytes: 4309
}

2022-01-20 16:27:47,428 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2022-01-20 16:27:47,533 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2022-01-20 16:27:47,635 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2022-01-20 16:27:47,737 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2022-01-20 16:27:47,880 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}

2022-01-20 16:27:48,324 INFO    MainThread:6724 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4309
  total_bytes: 4309
}
local_info {
}

2022-01-20 16:27:49,451 INFO    MainThread:6724 [wandb_run.py:_append_files():2184] logging synced files
